.github/renovate.json5

```diff
@@ -511,5 +511,16 @@
         "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\s+.+_VERSION:=(?<currentValue>.*)\\s+.+_DIGEST:=(?<currentDigest>sha256:[a-f0-9]+)"
       ]
     },
+    {
+      "fileMatch": [
+        ".github/actions/unit-tests/prepare.sh",
```

Comment:
I think we could just reuse the rexisting entry at https://github.com/cilium/cilium/pull/31685/files#diff-20ab1190d08a53a3012bc191ed56205c8f0ffb8fa1715e9d36ecfd1d2ea8a790L450-L466 for this file as well (i.e. just add `.github/actions/unit-tests/prepare.sh` to on line 454. And then you'll need to add an entry for the file to the `includePaths` section at the top of `.github/renovate.json5` (or extend the glob patterns) as well.